### PR TITLE
apps: Support setting CORS policies.

### PR DESCRIPTION
**Pull request by andrewsomething**
~Opening this as a draft for now as it still needs docs and more tests.~

Related to: https://github.com/digitalocean/terraform-provider-digitalocean/issues/686

### CODE REVIEW

**Comment by atombrella** [comment]
I had started a bit with this, but couldn't get the tests (`flatten` and `expand` problems) working. However, maybe copy-pasting the docs? https://github.com/atombrella/terraform-provider-digitalocean/commit/f8dd074312981f53ef6dabdf8c8b4740f4448f49#diff-2a77bc4b1d9ab16d12a37ea007beade1793d351c331bbb132899ffd61aa17efa

**Review by scotchneat — DISMISSED** [review]
Looks great. I just have one question about the tests but it's not a blocker.

File: digitalocean/resource_digitalocean_app_test.go [hunk 1]
```diff
@@ -586,6 +586,78 @@ func TestAccDigitalOceanApp_DomainsDeprecation(t *testing.T) {
...
+		appName, allowedOrgin,
+	)
+	updatedConfig := fmt.Sprintf(testAccCheckDigitalOceanAppConfig_CORS,
+		appName, fullConfig,
+	)
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckDigitalOceanAppDestroy,
+		Steps: []resource.TestStep{
```
[the remark this answers is not in the document]
Comment (andrewsomething) [reply digitalocean/resource_digitalocean_app_test.go:646]: Good call. Added in https://github.com/digitalocean/terraform-provider-digitalocean/pull/699/commits/63b96b2edf53261d68070d6b7381937b43c823a5

**Review by scotchneat — APPROVED** [review]
LGTM

**Pull request merged**